components/ai_chat/core/browser/associated_archive_content.h

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_COMPONENTS_AI_CHAT_CORE_BROWSER_ASSOCIATED_ARCHIVE_CONTENT_H_
+#define BRAVE_COMPONENTS_AI_CHAT_CORE_BROWSER_ASSOCIATED_ARCHIVE_CONTENT_H_
+
+#include <string>
+
+#include "base/memory/weak_ptr.h"
+#include "brave/components/ai_chat/core/browser/conversation_handler.h"
+#include "url/gurl.h"
+
+namespace ai_chat {
+
+class AssociatedArchiveContent
```

Comment:
A comment about exactly what this is for would be handy :smile:

Reply:
Added 8c401ad7b3bf8f2df987571698d4e0e4fc9256d2